users/tests/test_models.py

```diff
@@ -251,8 +253,12 @@ def test_user_serialize(snapshot, project):
         child_with_one_enrolment,
         child_without_enrolments,
     ] = ChildWithGuardianFactory.create_batch(3, relationship__guardian=guardian)
-    FreeSpotNotificationSubscriptionFactory(child=child_without_enrolments)
-    FreeSpotNotificationSubscriptionFactory(child=child_with_one_enrolment)
+    FreeSpotNotificationSubscriptionFactory(
+        child=child_without_enrolments, occurrence__id=101
+    )
+    FreeSpotNotificationSubscriptionFactory(
+        child=child_with_one_enrolment, occurrence__id=102
```

Comment:
What happens here? Can you override the ID when saving to the database like this or does the database generate the ID value and return it nonetheless if you try to override it? If you need a specific occurrence then why not create it first and reference it? Why the direct ID usage?

Reply:
The ID is there to fix the snapshot testing. The ID would change if the tests are run in a different order or only a subset would be ran.